custom/argo-workflow-executor/buildscripts/push

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+set -e
+
+IMAGEID=$( docker images -q litmuschaos/argoexec )
+
+if [ ! -z "${DNAME}" ] && [ ! -z "${DPASS}" ];
+then
+  docker login -u "${DNAME}" -p "${DPASS}";
+  #Push to docker hub repository with latest tag
```

Comment:
do we need this comment in all files? we are not pushing with latest tag.
OR should we push with the latest tag?

Reply:
It's already there in buildscript/push